.github/workflows/component_linux_unit_test.yml

```diff
@@ -20,6 +20,27 @@ jobs:
       - name: Validate code
         run: make ci/validate
 
+  run-lint:
+    name: Lint tests
+    continue-on-error: true
+    runs-on: ubuntu-20.04
```

Comment:
Should we bump ubuntu runner to the latest? 🤔

Reply:
Let's do that 👍